src/Microsoft.Identity.Client/Internal/Requests/RequestBase.cs

```diff
@@ -339,7 +339,9 @@ await AuthenticationRequestParameters
             client.AddBodyParameter(OAuth2Parameter.Scope,
                 GetDecoratedScope(AuthenticationRequestParameters.Scope).AsSingleString());
 
-            foreach (var kvp in additionalBodyParameters)
+            client.AddQueryParameter(OAuth2Parameter.Claims, AuthenticationRequestParameters.Claims);
+
+            foreach (KeyValuePair<string, string> kvp in additionalBodyParameters)
```

Comment:
aside: do we have team rules for var usage?  I don't find this extra explicitness here all that valuable and prefer var.  but happy to have the discussion.

Reply:
Agreed, I think my automated cleanup does too much with `var`. I'll revert this.